README.md

```diff
@@ -6,7 +6,7 @@ This is a development version of Configlet for use with v3 of Exercism.
 
 The application is a single binary and can be used as follows:
 
-```
+```console
```

Comment:
````suggestion
```shell
````

I think this is the language, but might be shellscript or bash.

Reply:
`console` is an alias for a shell *session* (https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5689) - i.e. a block that includes the prompt and output.

`shell` or `bash` is used for actual scripts (https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5602) - i.e. that would go into a script file

Reply:
Oh today I learned. Very nice.

Reply:
To be fair, I don't think either of them provide very good syntax highlighting for this purpose on GitHub (at least in my browser).

Reply:
`shell` or `console` preferred over `bash` unless that is the specific shell being used, the other two are "generic".

Reply:
Handled markdown linting separately in #488.

For now, we went with `text` instead of `console` here (because we don't currently show the command that produces the output).